packages/backend/src/scheduler/SchedulerClient.ts

```diff
@@ -52,7 +52,7 @@ type SchedulerClientArguments = {
     schedulerModel: SchedulerModel;
 };
 
-const SCHEDULED_JOB_MAX_ATTEMPTS = 1;
+const SCHEDULED_JOB_MAX_ATTEMPTS = 3;
```

Comment:
I would keep this as 1, as it might cause side effects on other scheduled deliveries.

Reply:
Let's increase the `maxRetries` in the emailClient to 3 and trigger the recreateTransporter on the last attempt (instead of hardcode second attempt). What do you think?